src/api/treeTrackerApi.js

```diff
@@ -130,12 +131,15 @@ export default {
    * Capture Match Tool
    */
   fetchCapturesToMatch(currentPage, abortController, filter) {
-    const where = Object.keys(filter)
-      .map((key) => (filter[key] ? `${key}=${filter[key]}` : ''))
-      .join('&');
+    const where = Object.keys(filter).reduce(
+      (str, key) => (filter[key] ? (str += `&${key}=${filter[key]}`) : str),
+      ''
+    );
+
     const req = `${CAPTURE_MATCH_API}/captures?tree_associated=false&limit=${1}&offset=${
       currentPage - 1
     }&${where}`;
```

Comment:
In the refactored `where` construction, there will always be a `&` at the beginning of the string already.

Reply:
I'll check this solution again, thanks!